code/modules/mob/abstract/new_player/menu.dm

```diff
@@ -227,10 +223,11 @@
 	if(SSticker.current_state <= GAME_STATE_PREGAME) // Make sure we don't ready up after the round has started
 		// Cannot join without a saved character, if we're on SQL saves.
 		if (config.sql_saves && !client.prefs.current_character)
-			alert(src, "You have not saved your character yet. Please do so before readying up.")
-			return
```

Comment:
The removal of the return here seems unintentional

Reply:
As discussed, it is not.